python/pyraphtory/tests/test_pyraphtory.py

```diff
@@ -78,11 +78,11 @@ def test_run_page_rank_algo(self):
     def test_check_vertex_properties_with_history(self):
         graph = self.ctx.new_graph()
         graph.add_vertex(1, 1, Properties(
-            ImmutableProperty("name", "TEST"),
-            StringProperty("strProp", "TESTstr"),
+            ImmutableString("name", "TEST"),
```

Comment:
To do review it to be a normal string in the future, with an immutable boolena flag?